Automattic/wp-calypso
# Jetpack Manage: Retire Partner portal 'Prices' page.

This page has become outdated and redundant with the introduction of license bundles and the plan to add a higher level **'Pricing'** tab in the sidebar. 

<img width="2552" alt="Screen Shot 2024-01-17 at 4 31 49 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/5953a7cb-cd40-4853-a390-91290c555a84">


Closes https://github.com/Automattic/jetpack-manage/issues/234

## Proposed Changes

* Remove the **'Purchases > Prices '** menu item in the sidebar.
* Remove `/partner-portal/prices` as a valid URL and remove related codes (context, components, etc.)


## Testing Instructions
Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb.

* Use the Jetpack cloud live link below and Go to the Overview page (`/overview`)
* Confirm that in the sidebar, the **'Purchases > Prices'** menu is no longer visible.
* Confirm that when accessing `/partner-portal/prices`, the page no longer loads.


## Pre-merge Checklist
